backend/src/main/java/de/bund/digitalservice/ris/search/controller/api/LiteratureController.java

```diff
@@ -93,17 +89,9 @@ public ResponseEntity<String> getLiteratureAsHtml(
           @Parameter(
               description =
                   "Used to select a different prefix for referenced resources, like images. Selecting 'PROXY' will prepend `/api`. Otherwise, the API base URL will be used.")
-          ResourceReferenceMode resourceReferenceMode)
-      throws ObjectStoreServiceException {
-    final String resourcePath = getResourceBasePath(resourceReferenceMode, documentNumber);
-    Optional<byte[]> bytes = literatureService.getFileByDocumentNumber(documentNumber);
-
-    if (bytes.isPresent()) {
-      String html = xsltTransformerService.transformCaseLaw(bytes.get(), resourcePath);
-      return ResponseEntity.ok(html);
-    } else {
-      return ResponseEntity.notFound().build();
-    }
+          ResourceReferenceMode resourceReferenceMode) {
+    // implement this correctly when the literature xslt transformer is available
+    return ResponseEntity.notFound().build();
```

Comment:
It is big nitpickying but I would throw a not-implemented status exception instead

`throw new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED, "Not implemented yet");`

Didn't check though is there is a controller advice transforming unexpected errors into something unrecognizable